apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -1675,6 +1689,21 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
                 .find((subAttribute: string) => typeof subAttribute === "string");
     };
 
+    /**
+     * Checks if the primary claim is verified.
+     *
+     * @param claimKey - The claim key to check.
+     * @returns True if the primary claim is verified, false otherwise.
+     */
+    const isPrimaryClaimVerified = (claimKey: string): boolean => {
```

Comment:
shall we move this method to utils/profile-utils.ts?

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/8137/commits/e8c612c9d1e6faea387ac942e84ad59d061f85f2